### ISSUE TITLE
Don't panic if no public ip found in master provisioning

### ISSUE THREAD

**Issue by jnummelin** [kontena/kontena#761]
Currently CLI panics if it cannot get public IP for (at least) master while waiting for provisioning on AWS. Might affect other providers as well

```
/Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/excon-0.49.0/lib/excon/connection.rb:89:in `initialize': no implicit conversion of nil into String (TypeError)
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/excon-0.49.0/lib/excon.rb:127:in `new'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/excon-0.49.0/lib/excon.rb:127:in `new'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/kontena-cli-0.13.4/lib/kontena/machine/aws/master_provisioner.rb:86:in `run!'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/kontena-cli-0.13.4/lib/kontena/cli/master/aws/create_command.rb:27:in `execute'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'
    from /Users/some_user/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/kontena-cli-0.13.4/bin/kontena:63:in `<top (required)>'
    from /Users/some_user/.rbenv/versions/2.3.1/bin/kontena:23:in `load'
    from /Users/some_user/.rbenv/versions/2.3.1/bin/kontena:23:in `<main>'
```

**Comment by jakolehm** [comment]
Yeah, cli should probably try private address (or atleast report that)

**Comment by jnummelin** [comment]
Fixed in #771
